src/main/java/racingcar/Car.java

```diff
@@ -0,0 +1,21 @@
+package racingcar;
+
+public class Car {
+
+    private static final int MOVABLE_COUNT = 4;
+    private int position;
```

Comment:
상수와 인스턴스 변수를 구분하게 쉽도록 공백 라인을 추가해보는 건 어떨까요?
```suggestion
    private static final int MOVABLE_COUNT = 4;
    
    private int position;
```